src/elements/Label/Readme.md

```diff
@@ -0,0 +1,26 @@
+### Usage
+```js static
+import { Label } from '@codespec/react-toolset'
+```
+
+### Basic Usage
+```js
+<Label>My Label</Label>
+```
+
+### Required Field Label
+```js
+<Label required={true}>
+  Required Label:
+</Label>
+```
+
+### Label For
+```js
+<div>
+  <Label htmlFor="simple-button">
```

Comment:
`htmlFor` is real react attribute for `for`, so I think we need to use another prop name for it.